src/main/java/frc/robot/EagleUtil.java

```diff
@@ -476,6 +491,26 @@ public static Pose2d getClosestLeftReef(Pose2d pose) {
     }
   }
 
+  public static Pose2d getClosestLeftReefBack(Pose2d pose) {
+    int closestReef = findClosestReefIndex(pose);
+    if (closestReef % 2 > 0) {
+      closestReef -= 1;
+    }
+
+    if (isRedAlliance()) {
+      calculateRedReefSetPoints();
+      Pose2d targetPose = redPoses[closestReef];
+      Rotation2d targetRotation = new Rotation2d(targetPose.getRotation().getRadians() + Math.PI);
```

Comment:
The main difference between this and the other path is the way we pick the target pose.  Don't duplicate the logic.
Instead, choose the target pose in the if/else block, then put shared logic afterwards